e2e/libs/volume/crd.py

```
@@ -246,6 +246,17 @@ def wait_for_restore_required_status(self, volume_name, restore_required_state):
             f"but got {volume['status']['restoreRequired']}\n"
 
+    def wait_for_volume_to_be_created(self, volume_name):
+        for i in range(self.retry_count):
+            logging(f"Waiting for volume {volume_name} to be created ... ({i})")
+            try:
+                self.get(volume_name)
+                return
+            except Exception as e:
+                print(f"Failed to wait for volume {volume_name} to be created: {e}")
+            time.sleep(self.retry_interval)
+        assert False, f"Failed to wait for volume {volume_name} to be created"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)